posthog/hogql/printer.py

```diff
@@ -491,6 +494,7 @@ def visit_join_expr(self, node: ast.JoinExpr) -> JoinExprResponse:
                 self.dialect == "clickhouse"
                 and not isinstance(table_type.table, FunctionCallTable)
                 and not isinstance(table_type.table, SavedQuery)
+                and not isinstance(table_type.table, ExchangeRateTable)
```

Comment:
Gotta skip adding the `team_id` clause to queries against this table or else it breaks catastrophically